generic_petg.xml.fdm_material

```diff
@@ -20,20 +20,26 @@ Generic PETG profile. The data in this file may not be correct for your specific
         <diameter>2.85</diameter>
     </properties>
     <settings>
-        <setting key="anti ooze retract position">-4</setting>
+        <!-- Deprime settings -->
+        <setting key="anti ooze retract position">0</setting>
         <setting key="anti ooze retract speed">5</setting>
-        <setting key="break preparation position">-16</setting>
-        <setting key="break preparation speed">50</setting>
+        <setting key="break preparation position">0</setting>
+        <setting key="break preparation speed">2</setting>
         <setting key="break preparation temperature">245</setting>
         <setting key="break position">-50</setting>
-        <setting key="break speed">25</setting>
-        <setting key="break temperature">75</setting>
-        <setting key="maximum park duration">7200</setting>
-        <setting key="no load move factor">0.95</setting>
-        <setting key="flush purge speed">0.5</setting>
-        <setting key="end of filament purge speed">0.5</setting>
+        <setting key="break speed">50</setting>
+        <setting key="break temperature">215</setting>
+        <setting key="pressure release dwell time">25</setting>
+        <setting key="dwell time before break preparation move">0</setting>
+        <setting key="end of print purge volume">0</setting>
+        <setting key="end of filament purge volume">0</setting>
         <setting key="flush purge length">60</setting>
```

Comment:
In MISP-1335 we also removed this property. Is it correct to keep it here?

Reply:
You mean the flush purge length? In the end we kept that property because of compatibility issues, right? At least this property is still there for the ultimaker profiles

Reply:
Okay, just making sure